api/alert_channels_jira_cloud_server.go

```diff
@@ -45,7 +73,8 @@ type JiraDataV2 struct {
 	JiraUrl            string `json:"jiraUrl"`
 	ProjectID          string `json:"projectId"`
 	Username           string `json:"username"`
-	Password           string `json:"password,omitempty"` // used for Jira Server
+	Password           string `json:"password,omitempty"`            // used for Jira Server
+	Configuration      string `json:"bidirectionalConfig,omitempty"` // used for bidirectional integration
```

Comment:
When empty this defaults to unidirectional?

Reply:
@dmurray-lacework , we should be good here....
<img width="804" alt="Screen Shot 2022-11-01 at 11 46 50 AM" src="https://user-images.githubusercontent.com/7862144/199289441-5b44db33-f469-42d5-9019-d0884da0f253.png">